src/test/java/wooteco/subway/admin/acceptance/LineStationAcceptanceTest.java

```diff
@@ -25,26 +39,134 @@ public static RequestSpecification given() {
     }
 
     /**
-     *     Given 지하철역이 여러 개 추가되어있다.
-     *     And 지하철 노선이 추가되어있다.
-     *
-     *     When 지하철 노선에 지하철역을 등록하는 요청을 한다.
-     *     Then 지하철역이 노선에 추가 되었다.
-     *
-     *     When 지하철 노선의 지하철역 목록 조회 요청을 한다.
-     *     Then 지하철역 목록을 응답 받는다.
-     *     And 새로 추가한 지하철역을 목록에서 찾는다.
-     *
-     *     When 지하철 노선에 포함된 특정 지하철역을 제외하는 요청을 한다.
-     *     Then 지하철역이 노선에서 제거 되었다.
-     *
-     *     When 지하철 노선의 지하철역 목록 조회 요청을 한다.
-     *     Then 지하철역 목록을 응답 받는다.
-     *     And 제외한 지하철역이 목록에 존재하지 않는다.
+     * Given 지하철역이 여러 개 추가되어있다.
+     * And 지하철 노선이 추가되어있다.
+     * <p>
+     * When 지하철 노선에 지하철역을 등록하는 요청을 한다.
+     * Then 지하철역이 노선에 추가 되었다.
+     * <p>
+     * When 지하철 노선의 지하철역 목록 조회 요청을 한다.
+     * Then 지하철역 목록을 응답 받는다.
+     * And 새로 추가한 지하철역을 목록에서 찾는다.
+     * <p>
+     * When 지하철 노선에 포함된 특정 지하철역을 제외하는 요청을 한다.
+     * Then 지하철역이 노선에서 제거 되었다.
+     * <p>
+     * When 지하철 노선의 지하철역 목록 조회 요청을 한다.
+     * Then 지하철역 목록을 응답 받는다.
+     * And 제외한 지하철역이 목록에 존재하지 않는다.
      */
     @DisplayName("지하철 노선에서 지하철역 추가 / 제외")
     @Test
     void manageLineStation() {
+        //given
+        createStation("강남");
+        createStation("서초");
+        createStation("구의");
+        createLine("2호선");
+        //when
+        register(getStations(), 1L);
+        LineResponse line = getLine(1L);
+        //then
+        assertThat(line.getStations().size()).isEqualTo(3);
+        //when
+        final Station station = line.getStations().get(0);
+        //then
+        assertThat(station.getName()).isEqualTo("강남");
+        //when
+        final int statusCode = deleteEdge(line.getId(), line.getStations().get(0).getId());
+        //then
+        assertThat(statusCode).isEqualTo(204);
```

Comment:
```suggestion
        assertThat(statusCode).isEqualTo(HttpStatus.CREATED.value());
```

매직넘버를 제거할 수 있는 부분은 최대한 제거해주시는 것이 가독성에 좋습니다